hoomd/update/box_resize.py

```diff
@@ -0,0 +1,76 @@
+from hoomd.operation import _Updater
+from hoomd.box import Box
+from hoomd.parameterdicts import ParameterDict
+from hoomd.typeconverter import OnlyType
+from hoomd.variant import Variant
+from hoomd.util import variant_preprocessing
+from hoomd import _hoomd
+
+
+class BoxResize(_Updater):
+    """Resizes the box between an initial and final box.
```

Comment:
So are each of the box lengths scaled linearly between the initial and final boxes, so that the volume doesn't linearly interpolate between the initial and final boxes?

I imagine it would be a useful and wanted feature to be able to linearly scale the volume between two boxes.

Reply:
It is scaled according to the variant provided to it. The `Variant` class can be inherited in Python so custom variants can be used. There are multiple out of the box variants as well not just linear.